data/games/bannedGames/saudi_arabia/bannedList.json

```
@@ -0,0 +1,69 @@
+{
+  "description":"A list of video games banned in Saudi Arabia",
+  "games":[
+    {
+    "name":"God of War (franchise)",
+    "reason":"Banned for a few weeks due to the presence of interactive sex scenes, partial nudity and sexual themes, as well as the word "God" in the title. In 2018, the newest game in the series was approved for release."
```

Fix a quote error:

```suggestion
    "reason":'Banned for a few weeks due to the presence of interactive sex scenes, partial nudity and sexual themes, as well as the word "God" in the title. In 2018, the newest game in the series was approved for release.'
```

https://travis-ci.org/dariusk/corpora/builds/660543011

---

Sorry, that also didn't work: https://travis-ci.org/dariusk/corpora/builds/660625599

Maybe try it like: `"reason":"......'...'......"`

Or escape the double quotes:  `"reason":"......\"...\"......"`